client/reader/search-stream/index.jsx

```diff
@@ -189,9 +190,11 @@ class SearchStream extends React.Component {
 							<FollowButton
 								followLabel={ translate( 'Follow %s', {
 									args: queryWithoutProtocol,
+									comment: '%s is the site name being followed. For example: "Discover"',
```

Comment:
Nit: might be my English, but this feels a bit ambiguous. Should it be `name of the site being followed` instead of `site name being followed`?

Reply:
Thanks @tyxla - your version is clearer! I'll update it before 🚢.